src/test/resources/application-flagsmith-test.properties

```diff
@@ -0,0 +1,3 @@
+terra.common.flagsmith.enabled=true
+terra.common.flagsmith.server-side-api-key=ser.Vn8MpGpej26aShW7CXsEaR
+terra.common.flagsmith.api-url=https://terra-flagsmith.api.verily.com/api/v1/
```

Comment:
Are these ok to check into terra-common-lib? And is it ok for this upstream repo to call the internal-env instance?

Reply:
i created a test env but now I moved it to use the public one.